MIGRATION.md

```diff
@@ -1,5 +1,14 @@
 # Migration Guide
 
+## Migration to Version ^8.0.0
+Version 8 introduces major updates of several libraries:
+- `commander`
+- `fs-extra`. This update fixes an issue where the `Layout.js` filename was being changed after a copy. This library is highly dependent on the Node version and was very out-of-date with respect to the current versions of Node, which  corrected this issue.
+- `glob`
+- `pluralize`
+
+There should should be no manual migration required by the user due to these changes.
```

Comment:
Can we lead with this?

Reply:
If there is no change needed on the user's end, is it really a breaking change? 🤔

Reply:
Maybe not. None of the major upgrades required any changes to the code. And, in one case (the strange problem with the `Layout.js` file's case becoming `layout.js`), the issue went away with no other change made by me. I'll revert to a minor update.